behat/tests/features/prof/03-report/09-deputy-discharged.feature

```diff
@@ -1,214 +0,0 @@
-Feature: Prof deputy is discharged
```

Comment:
Can this file be deleted rather than emptied?

Very happy to see old tests go 💪

Reply:
I thought I had deleted it. Resolved now